### PR TITLE
Update authors on labs

### PR DESCRIPTION
**Pull request by monaawi**
#1347
Missing things: hidden field for advisors, experiments etc.(schema, builders, loaders, ui), can_edit pop-up TBD with Stella, tests

### CODE REVIEW

File: ui/src/submissions/authors/containers/AuthorUpdateSuccessPageContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { AUTHORS } from '../../../common/routes';
+
+export function AuthorUpdateSuccessPage({ recordId }) {
+  return (
+    <SubmissionSuccess
+      message={
+        <span>
+          Successfully submitted, thank you! See the author profile{' '}
+          <Link to={`${AUTHORS}/${recordId}`}>here</Link>.
+            All proposed updates are reviewed by INSPIRE and further updates might be necessary to insure the best performance of the INSPIRE database.
```
Comment (harunurhan) [inline ui/src/submissions/authors/containers/AuthorUpdateSuccessPageContainer.jsx:15]: `insure` => `ensure` typo

File: ui/src/submissions/index.jsx [hunk 2]
```diff
@@ -78,7 +79,7 @@ class Submissions extends Component {
             <Redirect
```
Comment (harunurhan) [inline ui/src/submissions/index.jsx:79]: Must be `Route`

File: backend/inspirehep/submissions/views.py [hunk 3]
```diff
@@ -105,7 +105,40 @@ def post(self):
...
+        record.update(updated_record_data)
+        db.session.commit()
+
+        if not is_superuser_or_cataloger_logged_in():
+            self.create_ticket(record, "rt/update_author.html")
+
+        return jsonify({"pid_value": record["control_number"]})
+
+    def get_updated_record_data(self, update_data, record):
+        deletable_fields = [
```
Comment (harunurhan) [inline backend/inspirehep/submissions/views.py:129]: these should contain all non `required` fields.

File: backend/inspirehep/submissions/views.py [hunk 4]
```diff
@@ -105,7 +105,40 @@ def post(self):
...
+        updated_record_data = self.get_updated_record_data(data, record)
+        record.update(updated_record_data)
+        db.session.commit()
+
+        if not is_superuser_or_cataloger_logged_in():
+            self.create_ticket(record, "rt/update_author.html")
+
+        return jsonify({"pid_value": record["control_number"]})
+
+    def get_updated_record_data(self, update_data, record):
```
Comment (harunurhan) [inline backend/inspirehep/submissions/views.py:128]: I think almost identical logic exist elsewhere.

File: ui/src/submissions/index.jsx [hunk 5]
```diff
@@ -78,7 +79,7 @@ class Submissions extends Component {
...
               exact
               from={`${SUBMISSIONS_AUTHOR}/:id/success`}
```
Comment (harunurhan) [inline ui/src/submissions/index.jsx:81]: `path={...}`

**Comment by drjova** [comment]
Supersede by https://github.com/inspirehep/inspirehep/pull/1527

**Pull request closed without merging**